### PR TITLE
Inventory testing

### PR DESCRIPTION
**Pull request by aki2000ful**
inventory tests now pass

### CODE REVIEW

File: src/test/java/InventorySystem/TestInventory.java [hunk 1]
```diff
@@ -65,10 +69,12 @@ void testDamage() {
...
+        ArrayList<Item> list = new ArrayList<>();
+        list.add(sword);
+        list.add(dagger);
+        test_player.setInventory(list);
 
-        ArrayList<Item> inventory = characterManager.openInventory(test_player.id);
+        ArrayList<Item> inventory = test_player.getInventory();
         ArrayList<Item> comparison = new ArrayList<>();
         comparison.add(sword);
         comparison.add(dagger);
```
Comment (roylin2000) [inline src/test/java/InventorySystem/TestInventory.java]: I think you forgot to add an assert here.

**Review by roylin2000 — CHANGES_REQUESTED** [review]
Next time, you can merge this into your refactoring branch first since some changes here look identical to that. I'm not sure about the most proper process. Other than my comment in the code, it looks  good!

File: src/test/java/InventorySystem/TestInventory.java [hunk 2]
```diff
@@ -65,10 +69,12 @@ void testDamage() {
     void testInventory() {
         Item sword = new Sword(1);
         Item dagger = new Dagger(2);
```
Comment (matrix-marker) [inline src/test/java/InventorySystem/TestInventory.java]: these could be global to the class instead of declaring the exact same items over and over

**Review by matrix-marker — CHANGES_REQUESTED** [review]
the tests `testDamage` and `testInventory` effectively end up testing the LIst wrapper methods in Character

we want to be testing that you can properly manage that list from the CharacterManager, but now the only method that does that is `testSelect` – we have `addInventory` and `removeInventory` etc in the manager, and testing those ensures that the whole implementation works

the 2 tests mentioned could be moved to smth like `testCharacter`, and `TestInventory` needs more tests of the use-case

**Review by matrix-marker — APPROVED** [review]
tests are much more effective

**Review by matrix-marker — APPROVED** [review]

**Pull request merged**